packages/react/src/ReactElement.js

```diff
@@ -290,6 +292,11 @@ export function cloneAndReplaceKey(oldElement, newKey) {
  * See https://reactjs.org/docs/react-api.html#cloneelement
  */
 export function cloneElement(element, config, children) {
+  invariant(
+    !(element === null || element === undefined),
+    'Cannot clone a null or undefined element.'
```

Comment:
Maybe something like `React.cloneElement(...): The argument must be a React element, but you passed %s.` and either null or undefined as second argument (invariant will interpolate it).

Reply:
Okay thanks that’s cool, I’m going to read up on how invariant works for the future for sure